[NumberTaskFragment - Landscape ] NullPointerException for BindingAdapters.bindTextWatcher when rotating

```
java.lang.RuntimeException: Failed to call observer method
                                                                                                    	at androidx.lifecycle.ClassesInfoCache$MethodReference.invokeCallback(ClassesInfoCache.java:232)
                                                                                                    	at androidx.lifecycle.ClassesInfoCache$CallbackInfo.invokeMethodsForEvent(ClassesInfoCache.java:199)
                                                                                                    	at androidx.lifecycle.ClassesInfoCache$CallbackInfo.invokeCallbacks(ClassesInfoCache.java:190)
                                                                                                    	at androidx.lifecycle.ReflectiveGenericLifecycleObserver.onStateChanged(ReflectiveGenericLifecycleObserver.java:40)
                                                                                                    	at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.jvm.kt:320)
                                                                                                    	at androidx.lifecycle.LifecycleRegistry.addObserver(LifecycleRegistry.jvm.kt:198)
                                                                                                    	at androidx.databinding.ViewDataBinding.setLifecycleOwner(ViewDataBinding.java:434)
                                                                                                    	at com.google.android.ground.ui.datacollection.tasks.number.NumberTaskFragment.onCreateTaskBody(NumberTaskFragment.kt:35)
                                                                                                    	at com.google.android.ground.ui.datacollection.tasks.AbstractTaskFragment$onViewCreated$$inlined$doOnAttach$1.onViewAttachedToWindow(View.kt:419)
                                                                                                    	at android.view.View.dispatchAttachedToWindow(View.java:22885)
                                                                                                    	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3496)
                                                                                                    	at android.view.ViewGroup.addViewInner(ViewGroup.java:5344)
                                                                                                    	at android.view.ViewGroup.addView(ViewGroup.java:5130)
                                                                                                    	at android.view.ViewGroup.addView(ViewGroup.java:5070)
                                                                                                    	at android.view.ViewGroup.addView(ViewGroup.java:5042)
                                                                                                    	at androidx.viewpager2.adapter.FragmentStateAdapter.addViewToContainer(FragmentStateAdapter.java:402)
                                                                                                    	at androidx.viewpager2.adapter.FragmentStateAdapter.placeFragmentInViewHolder(FragmentStateAdapter.java:331)
                                                                                                    	at androidx.viewpager2.adapter.FragmentStateAdapter.onViewAttachedToWindow(FragmentStateAdapter.java:276)
                                                                                                    	at androidx.viewpager2.adapter.FragmentStateAdapter.onViewAttachedToWindow(FragmentStateAdapter.java:67)
                                                                                                    	at androidx.recyclerview.widget.RecyclerView.dispatchChildAttached(RecyclerView.java:7556)
                                                                                                    	at androidx.recyclerview.widget.RecyclerView$5.addView(RecyclerView.java:860)
                                                                                                    	at androidx.recyclerview.widget.ChildHelper.addView(ChildHelper.java:107)
                                                                                                    	at androidx.recyclerview.widget.RecyclerView$LayoutManager.addViewInt(RecyclerView.java:8601)
                                                                                                    	at androidx.recyclerview.widget.RecyclerView$LayoutManager.addView(RecyclerView.java:8559)
                                                                                                    	at androidx.recyclerview.widget.RecyclerView$LayoutManager.addView(RecyclerView.java:8547)
                                                                                                    	at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1641)
                                                                                                    	at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1587)
                                                                                                    	at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:665)
                                                                                                    	at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4134)
                                                                                                    	at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:3851)
                                                                                                    	at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4404)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at androidx.viewpager2.widget.ViewPager2.onLayout(ViewPager2.java:527)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at androidx.constraintlayout.widget.ConstraintLayout.onLayout(ConstraintLayout.java:1873)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
                                                                                                    	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
                                                                                                    	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
                                                                                                    	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
2024-10-15 12:47:26.699 28107-28107 AndroidRuntime          com.google.android.ground            E  	at android.view.View.layout(View.java:25388) (Ask Gemini)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
                                                                                                    	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1891)
                                                                                                    	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1729)
                                                                                                    	at android.widget.LinearLayout.onLayout(LinearLayout.java:1638)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
                                                                                                    	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1891)
                                                                                                    	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1729)
                                                                                                    	at android.widget.LinearLayout.onLayout(LinearLayout.java:1638)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
                                                                                                    	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
                                                                                                    	at com.android.internal.policy.DecorView.onLayout(DecorView.java:784)
                                                                                                    	at android.view.View.layout(View.java:25388)
                                                                                                    	at android.view.ViewGroup.layout(ViewGroup.java:6466)
                                                                                                    	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:4703)
                                                                                                    	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3959)
                                                                                                    	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2836)
                                                                                                    	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:10145)
                                                                                                    	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1406)
                                                                                                    	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1415)
                                                                                                    	at android.view.Choreographer.doCallbacks(Choreographer.java:1015)
                                                                                                    	at android.view.Choreographer.doFrame(Choreographer.java:945)
                                                                                                    	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1389)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:959)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:100)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:232)
                                                                                                    	at android.os.Looper.loop(Looper.java:317)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8674)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886)
                                                                                                    Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method com.google.android.ground.ui.common.BindingAdapters.bindTextWatcher, parameter textWatcher
                                                                                                    	at com.google.android.ground.ui.common.BindingAdapters.bindTextWatcher(Unknown Source:7)
                                                                                                    	at com.google.android.ground.databinding.NumberTaskFragBindingImpl.executeBindings(NumberTaskFragBindingImpl.java:140)
                                                                                                    	at androidx.databinding.ViewDataBinding.executeBindingsInternal(ViewDataBinding.java:512)
                                                                                                    	at androidx.databinding.ViewDataBinding.executePendingBindings(ViewDataBinding.java:484)
                                                                                                    	at androidx.databinding.ViewDataBinding$OnStartListener.onStart(ViewDataBinding.java:1706)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at androidx.lifecycle.ClassesInfoCache$MethodReference.invokeCallback(ClassesInfoCache.java:222)
                                                                                                    	... 90 more
```

--- a/ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/number/NumberTaskFragment.kt
+++ b/ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/number/NumberTaskFragment.kt
@@ -32,8 +32,8 @@ class NumberTaskFragment : AbstractTaskFragment<NumberTaskViewModel>() {
 
   override fun onCreateTaskBody(inflater: LayoutInflater): View {
     val taskBinding = NumberTaskFragBinding.inflate(inflater)
-    taskBinding.lifecycleOwner = this
     taskBinding.viewModel = viewModel
+    taskBinding.lifecycleOwner = this
     return taskBinding.root
   }
 }

